opbeans/views.py

```diff
@@ -21,6 +24,27 @@
 logger = logging.getLogger(__name__)
 
 
+def maybe_dt(view_func):
+    # either calls the view, or randomly forwards the request to another opbeans service
+    def wrapped_view(request, *args, **kwargs):
+        other_services = os.environ.get("OPBEANS_SERVICES")
```

Comment:
What will happen if you don't get any value in OPBEANS_SERVICES?, shouldn't you disable DT?

Reply:
That's in effect what happens in the if clause a bit further down